test/mail_getmail.bats

```diff
@@ -0,0 +1,19 @@
+load 'test_helper/common'
+
+function setup_file() {
+  local PRIVATE_CONFIG
+  PRIVATE_CONFIG=$(duplicate_config_for_container .)
+
+  docker run -d --name mail_getmail \
+    -v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
+    -v "$(pwd)/test/test-files":/tmp/docker-mailserver-test:ro \
+    -e ENABLE_GETMAIL=1 \
+    --cap-add=NET_ADMIN \
+    -h mail.my-domain.com -t "${NAME}"
+
+  wait_for_finished_setup_in_container mail_getmail
+}
+
+function teardown_file() {
+  docker rm -f mail_getmail
+}
```

Comment:
This test isn't serving any purpose, but we can discuss that once everything else from review is resolved.

I am aware that the current `getmail6` project on Github has their own tests that uses `docker-mailserver`, so we could probably reference what they're doing.

Reply:
Yeah ... not really sure what I can be doing here.  I'll take a look at the `getmail6 ` tests and see if I can update.

Reply:
We'll discuss this once another maintainer reviews and this is all that is left :+1:

Reply:
I adjusteded the test template, you will now only need to fill in the test functions. A good orientation are the Rspamd tests (as they are fairly recent) when it comes to using helper functions. See `test/tests/parallel/set1/spam_virus/rspamd...`.